### DISCUSSION TITLE
on 0.22, i cannot authenticate as admin user, getting "The requested resource wasn't found"

### DISCUSSION THREAD

**Discussion by starascendin** [pocketbase/pocketbase#6110] [Q&A]
hello, i'm attempting to login as admin user using code below:

```
export async function initAuth(pb: PocketBase, adminUserEmail: string, adminUserPassword: string) {
    // console.log('initAuth', config.pocketbaseEmail, config.pocketbasePassword);
    // if ( && config.pocketbasePassword) {
        try {
            console.log('initiating auth... adminUserEmail:', adminUserEmail, 'adminUserPassword:', adminUserPassword);
            const authData = await pb.admins.authWithPassword(adminUserEmail, adminUserPassword);
            console.log('authData', authData);
        } catch (error) {
            console.log('Failed to authenticate with PocketBase:', error);
        }
    // }
}

```

but i keep getting
```
 url: 'http://localhost:8090/api/collections/_superusers/auth-with-password',
  status: 404,
  response: {
    code: 404,
    message: "The requested resource wasn't found.",
    data: {}
  },
```

This is done on JS sdk 0.22 and pb 0.22.
If i downgrade to JS sdk 0.21, then the above admin login would go through. Am i missing something here?

Thanks for your help!

**Comment by ganigeorgiev** [comment]
JS SDK v0.22+ is compatible only with PocketBase v0.23+ (see the [JS SDK CHANGELOG](https://github.com/pocketbase/js-sdk/blob/master/CHANGELOG.md)).

If you are using PocketBase `<= v0.22.x` then you need to pin your JS SDK version constraint to `<v0.22.0`.

**Reply by starascendin** [reply]
ooo got it, i'll try w/ pocketbase 0.23

**Reply by ganigeorgiev** [reply]
Be aware that PocketBase v0.23 comes with a lot of breaking changes and requires manual migration steps (especially if you use the hooks). You can find more details about the changes and links to the Go/JSVM upgrade guides in https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#v0230.
